roles/openshift_excluder/tasks/verify_upgrade.yml

```diff
@@ -0,0 +1,15 @@
+---
+# input variables
+# - repoquery_cmd
+# - openshift_upgrade_target
+- include: init.yml
+
+- include: verify_excluder.yml
+  vars:
+    #repoquery_cmd: repoquery_cmd
+    #openshift_upgrade_target: openshift_upgrade_target
+    excluder: "{{ item }}"
+  with_items:
+  - "{{ openshift.common.service_type }}-docker-excluder"
+  - "{{ openshift.common.service_type }}-excluder"
+  when: docker_excluder_on or openshift_excluder_on
```

Comment:
Given repositories provided to a customer will always ship both excluders, the assumption is valid. Unless, there is a plan to ship individual rpms in different repositories some of which may be enabled and some disabled. @sdodson can you prove me wrong?